src/tools/pub/cmd/pub-get.md

```diff
@@ -143,6 +143,17 @@ command-line argument, as mentioned above.
 For options that apply to all pub commands, see
 [Global options](/tools/pub/cmd#global-options).
 
+### `--dry-run` or `-n`
+
+With this, pub reports the dependencies that would
+but does not make the changes. This is useful if you
+want to analyze updates before making them.
+
```

Comment:
Let's add the `--offline` flag to this section. (It can mostly point to previous sections.)